app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,26 +1,40 @@
 import React, { useState, useEffect } from "react";
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Button, PageLoader } from "neetoui";
-import { Header, SubHeader } from "neetoui/layouts";
+import { Search } from "neetoicons";
+import { PageLoader } from "neetoui";
+import { Button, Input } from "neetoui/v2";
+import { Header } from "neetoui/v2/layouts";
 
 import notesApi from "apis/notes";
 import EmptyState from "components/Common/EmptyState";
+import Menubar from "components/Common/Menubar";
 
-import DeleteAlert from "./DeleteAlert";
 import NewNotePane from "./NewNotePane";
-import NoteTable from "./NoteTable";
+import NotesCard from "./NotesCard";
 
 const Notes = () => {
-  const [loading, setLoading] = useState(true);
+  const [loading, setLoading] = useState(false);
   const [showNewNotePane, setShowNewNotePane] = useState(false);
-  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+  const [showMenu, setShowMenu] = useState(true);
   const [searchTerm, setSearchTerm] = useState("");
-  const [selectedNoteIds, setSelectedNoteIds] = useState([]);
-  const [notes, setNotes] = useState([]);
+  const [notes, setNotes] = useState([
+    {
+      title: "How to claim the warranty?",
+      content: `"Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`
+    },
+    {
+      title: "How to claim the warranty?",
+      content: `"Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`
+    },
+    {
+      title: "How to claim the warranty?",
+      content: `"Are you getting my texts???" she texted to him. He glanced at it and chuckled under his breath. Of course he was getting them, but if he wasn't getting`
+    }
+  ]);
```

Comment:
Please move this content into a separate constants file called `constants.js`. It's a good practice to move constants to a separate file.